packages/mobile/src/components/core/Pressable.tsx

```diff
@@ -0,0 +1,52 @@
+import { useCallback, useRef } from 'react'
+
+import {
+  GestureResponderEvent,
+  Pressable as RNPressable,
+  PressableProps as RNPressableProps
+} from 'react-native'
+
+type PressableProps = RNPressableProps
+
+type TouchPosition = { pageX: number; pageY: number }
+
+export const Pressable = (props: PressableProps) => {
+  const { onPress, onPressIn, ...other } = props
+
+  const activeTouchPositionRef = useRef<TouchPosition | null>(null)
+
+  const handlePressIn = useCallback(
+    (event: GestureResponderEvent) => {
+      const { pageX, pageY } = event.nativeEvent
+
+      activeTouchPositionRef.current = {
+        pageX,
+        pageY
+      }
+
+      onPressIn?.(event)
+    },
+    [onPressIn]
+  )
+
+  const handlePress = useCallback(
+    (event: GestureResponderEvent) => {
+      if (!activeTouchPositionRef.current) return
+
+      const { pageX, pageY } = event.nativeEvent
+
+      const absX = Math.abs(activeTouchPositionRef.current.pageX - pageX)
+      const absY = Math.abs(activeTouchPositionRef.current.pageY - pageY)
+
+      const dragged = absX > 2 || absY > 2
```

Comment:
worth pulling 2 into a const so it's clear that it's the abs pos change for scroll?  separately kinda curious why not 1 px

Reply:
yeah no idea on why 2, that's what they used for the example, will extract and clarify the values here too, prob gonna use dx and dy instead of absX absY